qiskit/ignis/verification/tomography/basis/circuits.py

```diff
@@ -129,10 +131,102 @@ def process_tomography_circuits(
                                 prep_labels=prep_labels, prep_basis=prep_basis)
 
 
+###########################################################################
+# Gate set tomography circuits for preparation and measurement
+###########################################################################
+
+def gateset_tomography_circuits(gateset_basis='Standard GST'):
```

Comment:
I think this needs to be able to take a qubit argument so it can be run on specific qubits. Since we only support 1-qubit tomography for now, it would make sense to have the functionality of this function be a helper function (something like `gateset_tomography_circuits_1q`), and then have it called by a larger function like:

```python
def gateset_tomography_circuits(measured_qubits, gateset_basis='default'):
    # Raise error if measured_qubits is more than 1-qubit
    
    if measured_qubits == [0]:
        return gateset_tomography_circuits_1q(gateset_basis)

    # Otherwise determine number of qubits needed in circuit from arg
    # and apply the 1 qubit circuits to the specified qubit
    num_qubits = 1 + max(measured_qubits)
    circuits = []
    for circ_1q in gateset_tomography_circuits_1q(gateset_basis):
        circ = QuantumCircuit(num_qubits, name=circ_1q.name)
        circ.append(circ_1q, measured_qubits)
        circ = circ.decompose()  # unroll circuit instruction to the gates in the original circuit
        circuits.append(circ)
   return circuits
```
where qubits is a list of ints or qr (same as tomography), and we raise an exception if it is not 1-qubit since we only support 1 qubit for now.